README.md

```diff
@@ -98,8 +100,11 @@ Now we need to install any dependencies. The next step is executed from inside t
 sudo ansible-galaxy install -r requirements.yml
 ```
 
-Next we need to configure Docker to accept an insecure registry required as part of the cluster setup.
-For Linux follow steps 2 and 3 [here](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#linux) and the same for Mac [here](https://github.com/openshift/origin/blob/master/docs/cluster_up_down.md#macos-with-docker-for-mac)
+#### Docker Firewall setup
+
+Next we need to configure Docker to accept an insecure registry required as part of the cluster setup:
```

Comment:
Maybe indicate the ports as well here as I think you skipped the firewall parts since you already had the registry done?
Maybe:
`Next we need to configure Docker registry and ports required as part of the cluster setup:`?